src/_guides/language/language-tour.md

```diff
@@ -1553,25 +1540,22 @@ prints a string that includes the value at the specified index.
 
 <?code-excerpt "../null_safety_examples/misc/test/language_tour/functions_test.dart (anonymous-function)"?>
 ```dart
-var list = ['apples', 'bananas', 'oranges'];
+const list = ['apples', 'bananas', 'oranges'];
 list.forEach((item) {
   print('${list.indexOf(item)}: $item');
 });
 ```
 
 Click **Run** to execute the code.
 
-{% comment %}
-https://gist.github.com/chalin/5d70bc1889d055c7a18d35d77874af88
-{{site.dartpad}}/5d70bc1889d055c7a18d35d77874af88
-{% endcomment %}
-
-<iframe
-src="{{site.dartpad-embed-inline}}?id=5d70bc1889d055c7a18d35d77874af88&split=60&ga_id=anonymous_functions"
-    width="100%"
-    height="400px"
-    style="border: 1px solid #ccc;">
-</iframe>
+```dart:run-dartpad:height-400px:ga_id-anonymous_functions:null_safety-true
```

Comment:
Can we add a code-excerpt element above here?

Reply:
Good idea!